test/e2e/dra/dra.go

```diff
@@ -1004,11 +1004,18 @@ var _ = framework.SIGDescribe("node")("DRA", feature.DynamicResourceAllocation,
 						Used: v1.ResourceList{v1.ResourceName(resourceName): resource.MustParse("1")},
 					})})))
 
-			// Now creating another claim should fail.
-			claim2 := claim.DeepCopy()
-			claim2.Name = "claim-1"
-			_, err = f.ClientSet.ResourceV1beta1().ResourceClaims(f.Namespace.Name).Create(ctx, claim2, metav1.CreateOptions{})
-			gomega.Expect(err).Should(gomega.MatchError(gomega.ContainSubstring("exceeded quota: object-count, requested: count/resourceclaims.resource.k8s.io=1, used: count/resourceclaims.resource.k8s.io=1, limited: count/resourceclaims.resource.k8s.io=1")), "creating second claim not allowed")
+			// Now creating another claim should eventually fail. The quota may not be enforced immediately if
+			// it hasn't yet landed in the API server's cache.
+			gomega.Eventually(func(g gomega.Gomega) {
```

Comment:
I think if we're passing in `g gomega.Gomega` we would need to use `g` to invoke all the gomega foo within the Eventually func. We can probably just skip that and do something like this instead:

```golang
gomega.Eventually(func() err {
claim2 := claim.DeepCopy()
	// If creating a claim erroneously succeeds, we don't want to immediately fail on the next try
	// with an "already exists" error, so use a new name each time.
	claim2.GenerateName = "claim-1-"
	claim2.Name = ""

	_, err := f.ClientSet.ResourceV1beta1().ResourceClaims(f.Namespace.Name).Create(ctx, claim2, metav1.CreateOptions{})
	return err
}).Should(gomega.Expect(err).Should(gomega.MatchError(gomega.ContainSubstring("exceeded quota: object-count, requested: count/resourceclaims.resource.k8s.io=1, used: count/resourceclaims.resource.k8s.io=1, limited: count/resourceclaims.resource.k8s.io=1")), "creating second claim not allowed"))
```

Reply:
Yes, you're right. Thanks!

I'm less confident in that exact syntax, but I'll push the fix to use the `g` inside the `Eventually` now and see if what you had there works.

Reply:
Oh I think I see what you meant. I'll try that out and push that.